main.go

```diff
@@ -319,13 +320,26 @@ func awss3(w http.ResponseWriter, r *http.Request) {
 	setStrHeader(w, "Content-Type", obj.ContentType)
 	setTimeHeader(w, "Last-Modified", obj.LastModified)
 
+
+	if obj.ContentRange != nil && len(*obj.ContentRange) > 0 {
+		w.WriteHeader(http.StatusPartialContent)
+	}
+
 	io.Copy(w, obj.Body)
 }
 
-func s3get(backet, key string) (*s3.GetObjectOutput, error) {
+func s3get(backet, key, rangeHeader string) (*s3.GetObjectOutput, error) {
+	var rangeHeaderAwsString *string = nil
```

Comment:
__[Golint]__

should drop = nil from declaration of var rangeHeaderAwsString; it is the zero value


<!-- Comment by [SideCI](https://sideci.com) -->